pom.xml

```diff
@@ -55,11 +55,49 @@
 
   <dependencies>
     <dependency>
+     <groupId>org.jenkins-ci.main</groupId>
```

Comment:
It should not be done in such way. Effectively you bundle Jenkins core and its deps into a plugin, which may cause some inexpected results when a user installs the plugin.

If you need a specific core version, use

```
<properties>
  <jenkins.version>1.642.3</jenkins.version>
</properties>
```